ngraph/python/tests/test_ngraph/test_ops_reshape.py

```diff
@@ -125,13 +126,31 @@ def test_gather():
         [1.0, 1.1, 1.2, 2.0, 2.1, 2.2, 3.0, 3.1, 3.2], np.float32
     ).reshape((3, 3))
     input_indices = np.array([0, 2], np.int32).reshape(1, 2)
-    input_axes = np.array([1], np.int32)
+    input_axis = np.array([1], np.int32)
 
     expected = np.array([1.0, 1.2, 2.0, 2.2, 3.0, 3.2], dtype=np.float32).reshape(
         (3, 1, 2)
     )
 
-    result = run_op_node([input_data], ng.gather, input_indices, input_axes)
+    result = run_op_node([input_data], ng.gather, input_indices, input_axis)
+    assert np.allclose(result, expected)
+
+
+@xfail_issue_49062
+def test_gather_batch_dims_1():
+
```

Comment:
Also, could you please move gather related tests into separate file (do not use one for Reshape operation).

Reply:
done